frontends/p4/parseAnnotations.cpp

```diff
@@ -57,94 +57,89 @@ ParseAnnotations::HandlerMap ParseAnnotations::standardHandlers() {
 bool ParseAnnotations::parseSkip(IR::Annotation *) { return false; }
 
 bool ParseAnnotations::parseEmpty(IR::Annotation *annotation) {
-    if (!annotation->body.empty()) {
+    if (!annotation->getUnparsed().empty()) {
```

Comment:
Can we assume here the annotation is unparsed? If not, this will throw `std::bad_variant_access`. Probably we can as we are parsing annotations...

Reply:
Yeah, that's exactly the point. Duplicate parsing will throw exceptions